src/UnitTestUtilities/appsettingsUnitTest.json

```diff
@@ -1,5 +1,5 @@
 {
   "SW360URL": "http://localhost:8090",
   "FossologyURL": "http://localhost:8091",
-  "JFrogURL": ""
+  "JFrogURL": "https://siemens.jfrog.io/artifactory"
```

Comment:
remove siemens URLs.Major security issue

Reply:
By Mistake it added.